pkg/providers/kube/client_test.go

```diff
@@ -273,7 +272,7 @@ var _ = Describe("Test Kube Client Provider (w/o kubecontroller)", func() {
 			},
 		}, nil)
 
-		portToProtocolMap, err := provider.GetTargetPortToProtocolMappingForService(tests.BookbuyerService)
+		portToProtocolMap, err := endpointProvider.GetTargetPortToProtocolMappingForService(tests.BookbuyerService)
```

Comment:
Update based on if GetTargetPortToProtocolMappingForService should  stay apart of endpoint provider vs service provider